financialaid/api_tests.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Please rename this file to `api_test.py`. Our testing code only detects based on `*_test.py` currently